doc/user-guides/acrn-dm-parameters.rst

```diff
@@ -472,3 +453,178 @@ Here are descriptions for each of these ``acrn-dm`` command line parameters:
+       Serial ATA (SATA) storage devices, such as a hard disk. Parameter
+       ``<type:><filepath>*`` should be added: ``type`` could be
+       ``hd`` (harddisk) or ``cd`` (CD-ROM). ``<filepath>`` is the path for the
+       backend file and could be a partition name or a regular file, e.g.,
+       ``ahci,hd:/dev/sda``.
+
+   * - ``ahci-hd``
+     - This is an alias for ``ahci``.
+
+   * - ``ahci-cd``
+     - Advanced Host Controller Interface used to connect with AT Attachment
+       Packet Interface device (for CD-ROM emulation). ``ahci-cd`` has the same
+       parameters with ``ahci``.
+
+   * - ``amd_hostbridge``
+     - Virtualized PCI AMD hostbridge
+
+   * - ``hostbridge``
+     - Virtualized PCI hostbridge, a hardware bridge between the CPU's
+       high-speed system local bus and the Peripheral Component Interconnect
+       (PCI) bus.
+
+   * - ``virtio-blk``
+     - Virtio block type device, a string could be appended to specify the disk
+       image, e.g., ``virtio-blk,/root/test.img``.  You can also could use
+       ``nodisk`` to create a virtio-blk device with a dummy backend.
+
+   * - ``virtio-coreu``
+     - Used for Protected Audio Visual Path (PAVP) session management to provide
+       a User VM with Protected Audio Visual Path service.
+
+   * - ``virtio-input``
+     - Virtio type device to emulate input device. ``evdev`` char device node
+       should be appended, e.g., ``-s
+       n,virtio-input,/dev/input/eventX[,serial]``. ``serial`` is an optional
+       string used as the Uniq of guest virtio input device.
+
+   * - ``virtio-ipu``
+     - Virtio image processing unit (IPU) type device, such as a camera.
+
+   * - ``virtio-console``
+     - Virtio console type device for data input and output.
+
+   * - ``virtio-hyper_dmabuf``
+     - Virtio device that allows sharing data buffers between VMs using a
+       dmabuf-like interface.
+
+   * - ``virtio-hdcp``
+     - Virtio High-bandwidth Digital Content Protection (HDCP) type device. HDCP
+       is technology intended to protect unauthorized duplication of high
+       definition (HD) video and audio as it travels across connections.
+
+   * - ``virtio-heci``
+     - Virtio Host Embedded Controller Interface, parameters should be appended
+       with the format ``<bus>:<device>:<function>,d<0~8>``.
+
+   * - ``virtio-i2c``
+     - Virtio i2c type device,parameters with format:
+       ``<bus>[:<client_addr>[@<node>]][,<bus>[:<client_addr>[@<node>]]``
+
+       * ``<bus>`` specifies the bus number for the native I2C adapter, e.g.,
+         ``2`` means ``/dev/i2c-2``.
+       * ``<client_addr>`` specifies the address for the native client devices
+         such as ``1C`` or  ``2F``.
+       * ``@`` specifies the prefix for the ACPI node.
+       * ``<node>`` specifies the ACPI node name supported in the
+         ``acpi_node_table[]`` in the source code: only ``cam1``, ``cam2``, and
```

Comment:
I don't think this mention of the ``acpi_node_table`` (found in virtio_i2c.c) is needed since there's no further documentation provided there.   If these are the only three names supported, we can just list them and be done, though I'm curious about how additional supported node names are added since they're platform-specific and you mention only one platform ( ``MRB``) has these names defined).

I see there's one additional name in this table you've not mentioned and we should document:  ``default``

Reply:
which one I did not mention? bus/client_addr/@/node ?

Reply:
no additional supported node besides the three in APL platform